[Remote clusters] Update copy

alisonelizabeth opened the pull request:
This PR updates copy in the remote clusters UI.

Related to https://github.com/elastic/kibana/pull/59221

### Screenshots

<img width="1775" alt="Screen Shot 2020-03-17 at 10 41 19 AM" src="https://user-images.githubusercontent.com/5226211/76867875-8102eb00-683c-11ea-95b0-878659c42ed6.png">
<img width="1251" alt="Screen Shot 2020-03-17 at 10 41 35 AM" src="https://user-images.githubusercontent.com/5226211/76867877-819b8180-683c-11ea-897b-99fdebf4c49d.png">
<img width="1248" alt="Screen Shot 2020-03-17 at 10 42 08 AM" src="https://user-images.githubusercontent.com/5226211/76867878-819b8180-683c-11ea-95aa-9a18f6a1ca72.png">

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/)
* Commit: 79e41e31def49ccc85257edd1eb91a6067ba291a

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/testReport)

<details><summary>Kibana Pipeline / x-pack-intake-agent / X-Pack Jest Tests.x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form.RemoteClusterForm renders untouched state</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/testReport//junit/X-Pack%20Jest%20Tests/x-pack_plugins_remote_clusters_public_application_sections_components_remote_cluster_form/Kibana_Pipeline___x_pack_intake_agent___RemoteClusterForm_renders_untouched_state)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
Error: expect(received).toMatchSnapshot()

Snapshot name: `RemoteClusterForm renders untouched state 1`

- Snapshot
+ Received

@@ -26,11 +26,11 @@
              class="euiText euiText--small euiDescribedFormGroup__description"
            >
              <div
                class="euiTextColor euiTextColor--subdued"
              >
-               A unique name for the remote cluster.
+               A unique name for the cluster.
              </div>
            </div>
          </div>
          <div
            class="euiFlexItem euiDescribedFormGroup__fields euiDescribedFormGroup__fieldPadding--xsmall"
@@ -105,11 +105,11 @@
              class="euiText euiText--small euiDescribedFormGroup__description"
            >
              <div
                class="euiTextColor euiTextColor--subdued"
              >
-               Remote cluster connections work by configuring a remote cluster and connecting only to a limited number of nodes in that remote cluster.
+               Use seed nodes by default, or switch to a single proxy address.
                <div
                  class="euiFormRow euiFormRow--hasEmptyLabelSpace euiFormRow--fullWidth"
                  id="mockId-row"
                >
                  <div
@@ -118,11 +118,10 @@
                    <div
                      class="euiSwitch"
                    >
                      <button
                        aria-checked="false"
-                       aria-describedby="mockId-help"
                        aria-labelledby="mockId"
                        class="euiSwitch__button"
                        data-test-subj="remoteClusterFormConnectionModeToggle"
                        id="mockId"
                        role="switch"
@@ -150,25 +149,13 @@
                      </button>
                      <span
                        class="euiSwitch__label"
                        id="mockId"
                      >
-                       Use proxy mode
+                       Use a single proxy address
                      </span>
                    </div>
-                   <div
-                     class="euiFormHelpText euiFormRow__text"
-                     id="mockId-help"
-                   >
-                     Configure a remote cluster with a single proxy address. 
-                     <button
-                       class="euiLink euiLink--primary"
-                       type="button"
-                     >
-                       Learn more.
-                     </button>
-                   </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
@@ -322,11 +309,11 @@
            >
              <div
                class="euiTextColor euiTextColor--subdued"
              >
                <p>
-                 By default, a request fails if any of the queried remote clusters are unavailable. To continue sending a request to other remote clusters if this cluster is unavailable, enable 
+                 A request fails if any of the queried remote clusters are unavailable. To send requests to other remote clusters if this cluster is unavailable, enable 
                  <strong>
                    Skip if unavailable
                  </strong>
                  . 
                  <button
    at Object.test (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form/remote_cluster_form.test.js:21:23)
    at Object.asyncJestTest (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
```
</details>

---

<details><summary>Kibana Pipeline / x-pack-intake-agent / X-Pack Jest Tests.x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form.RemoteClusterForm proxy mode renders correct connection settings when user enables proxy mode</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/testReport//junit/X-Pack%20Jest%20Tests/x-pack_plugins_remote_clusters_public_application_sections_components_remote_cluster_form/Kibana_Pipeline___x_pack_intake_agent___RemoteClusterForm_proxy_mode_renders_correct_connection_settings_when_user_enables_proxy_mode)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
Error: expect(received).toMatchSnapshot()

Snapshot name: `RemoteClusterForm proxy mode renders correct connection settings when user enables proxy mode 1`

- Snapshot
+ Received

@@ -121,11 +121,11 @@
        className="euiForm"
      >
        <EuiDescribedFormGroup
          description={
            <FormattedMessage
-             defaultMessage="A unique name for the remote cluster."
+             defaultMessage="A unique name for the cluster."
              id="xpack.remoteClusters.remoteClusterForm.sectionNameDescription"
              values={Object {}}
            />
          }
          fullWidth={true}
@@ -204,15 +204,15 @@
                          >
                            <div
                              className="euiTextColor euiTextColor--subdued"
                            >
                              <FormattedMessage
-                               defaultMessage="A unique name for the remote cluster."
+                               defaultMessage="A unique name for the cluster."
                                id="xpack.remoteClusters.remoteClusterForm.sectionNameDescription"
                                values={Object {}}
                              >
-                               A unique name for the remote cluster.
+                               A unique name for the cluster.
                              </FormattedMessage>
                            </div>
                          </EuiTextColor>
                        </div>
                      </EuiText>
@@ -359,47 +359,28 @@
        </EuiDescribedFormGroup>
        <EuiDescribedFormGroup
          description={
            <React.Fragment>
              <FormattedMessage
-               defaultMessage="Remote cluster connections work by configuring a remote cluster and connecting only to a limited number of nodes in that remote cluster."
+               defaultMessage="Use seed nodes by default, or switch to a single proxy address."
                id="xpack.remoteClusters.remoteClusterForm.sectionModeDescription"
                values={Object {}}
              />
              <EuiFormRow
                describedByIds={Array []}
                display="row"
                fullWidth={true}
                hasChildLabel={true}
                hasEmptyLabelSpace={true}
-               helpText={
-                 <FormattedMessage
-                   defaultMessage="Configure a remote cluster with a single proxy address. {learnMoreLink}"
-                   id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription"
-                   values={
-                     Object {
-                       "learnMoreLink": <ForwardRef
-                         target="_blank"
-                       >
-                         <FormattedMessage
-                           defaultMessage="Learn more."
-                           id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription.learnMoreLinkLabel"
-                           values={Object {}}
-                         />
-                       </ForwardRef>,
-                     }
-                   }
-                 />
-               }
                labelType="label"
              >
                <EuiSwitch
                  checked={true}
                  data-test-subj="remoteClusterFormConnectionModeToggle"
                  label={
                    <FormattedMessage
-                     defaultMessage="Use proxy mode"
+                     defaultMessage="Use a single proxy address"
                      id="xpack.remoteClusters.remoteClusterForm.fieldModeLabel"
                      values={Object {}}
                    />
                  }
                  onChange={[Function]}
@@ -483,58 +464,38 @@
                          >
                            <div
                              className="euiTextColor euiTextColor--subdued"
                            >
                              <FormattedMessage
-                               defaultMessage="Remote cluster connections work by configuring a remote cluster and connecting only to a limited number of nodes in that remote cluster."
+                               defaultMessage="Use seed nodes by default, or switch to a single proxy address."
                                id="xpack.remoteClusters.remoteClusterForm.sectionModeDescription"
                                values={Object {}}
                              >
-                               Remote cluster connections work by configuring a remote cluster and connecting only to a limited number of nodes in that remote cluster.
+                               Use seed nodes by default, or switch to a single proxy address.
                              </FormattedMessage>
                              <EuiFormRow
                                describedByIds={Array []}
                                display="row"
                                fullWidth={true}
                                hasChildLabel={true}
                                hasEmptyLabelSpace={true}
-                               helpText={
-                                 <FormattedMessage
-                                   defaultMessage="Configure a remote cluster with a single proxy address. {learnMoreLink}"
-                                   id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription"
-                                   values={
-                                     Object {
-                                       "learnMoreLink": <ForwardRef
-                                         target="_blank"
-                                       >
-                                         <FormattedMessage
-                                           defaultMessage="Learn more."
-                                           id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription.learnMoreLinkLabel"
-                                           values={Object {}}
-                                         />
-                                       </ForwardRef>,
-                                     }
-                                   }
-                                 />
-                               }
                                labelType="label"
                              >
                                <div
                                  className="euiFormRow euiFormRow--hasEmptyLabelSpace euiFormRow--fullWidth"
                                  id="mockId-row"
                                >
                                  <div
                                    className="euiFormRow__fieldWrapper"
                                  >
                                    <EuiSwitch
-                                     aria-describedby="mockId-help"
                                      checked={true}
                                      data-test-subj="remoteClusterFormConnectionModeToggle"
                                      id="mockId"
                                      label={
                                        <FormattedMessage
-                                         defaultMessage="Use proxy mode"
+                                         defaultMessage="Use a single proxy address"
                                          id="xpack.remoteClusters.remoteClusterForm.fieldModeLabel"
                                          values={Object {}}
                                        />
                                      }
                                      onBlur={[Function]}
@@ -544,11 +505,10 @@
                                      <div
                                        className="euiSwitch"
                                      >
                                        <button
                                          aria-checked={true}
-                                         aria-describedby="mockId-help"
                                          aria-labelledby="mockId"
                                          className="euiSwitch__button"
                                          data-test-subj="remoteClusterFormConnectionModeToggle"
                                          id="mockId"
                                          onBlur={[Function]}
@@ -595,64 +555,19 @@
                                          className="euiSwitch__label"
                                          id="mockId"
                                          onClick={[Function]}
                                        >
                                          <FormattedMessage
-                                           defaultMessage="Use proxy mode"
+                                           defaultMessage="Use a single proxy address"
                                            id="xpack.remoteClusters.remoteClusterForm.fieldModeLabel"
                                            values={Object {}}
                                          >
-                                           Use proxy mode
+                                           Use a single proxy address
                                          </FormattedMessage>
                                        </span>
                                      </div>
                                    </EuiSwitch>
-                                   <EuiFormHelpText
-                                     className="euiFormRow__text"
-                                     id="mockId-help"
-                                   >
-                                     <div
-                                       className="euiFormHelpText euiFormRow__text"
-                                       id="mockId-help"
-                                     >
-                                       <FormattedMessage
-                                         defaultMessage="Configure a remote cluster with a single proxy address. {learnMoreLink}"
-                                         id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription"
-                                         values={
-                                           Object {
-                                             "learnMoreLink": <ForwardRef
-                                               target="_blank"
-                                             >
-                                               <FormattedMessage
-                                                 defaultMessage="Learn more."
-                                                 id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription.learnMoreLinkLabel"
-                                                 values={Object {}}
-                                               />
-                                             </ForwardRef>,
-                                           }
-                                         }
-                                       >
-                                         Configure a remote cluster with a single proxy address. 
-                                         <EuiLink
-                                           target="_blank"
-                                         >
-                                           <button
-                                             className="euiLink euiLink--primary"
-                                             type="button"
-                                           >
-                                             <FormattedMessage
-                                               defaultMessage="Learn more."
-                                               id="xpack.remoteClusters.remoteClusterForm.fieldModeDescription.learnMoreLinkLabel"
-                                               values={Object {}}
-                                             >
-                                               Learn more.
-                                             </FormattedMessage>
-                                           </button>
-                                         </EuiLink>
-                                       </FormattedMessage>
-                                     </div>
-                                   </EuiFormHelpText>
                                  </div>
                                </div>
                              </EuiFormRow>
                            </div>
                          </EuiTextColor>
@@ -1028,11 +943,11 @@
        <EuiDescribedFormGroup
          description={
            <React.Fragment>
              <p>
                <FormattedMessage
-                 defaultMessage="By default, a request fails if any of the queried remote clusters are unavailable. To continue sending a request to other remote clusters if this cluster is unavailable, enable {optionName}. {learnMoreLink}"
+                 defaultMessage="A request fails if any of the queried remote clusters are unavailable. To send requests to other remote clusters if this cluster is unavailable, enable {optionName}. {learnMoreLink}"
                  id="xpack.remoteClusters.remoteClusterForm.sectionSkipUnavailableDescription"
                  values={
                    Object {
                      "learnMoreLink": <ForwardRef
                        target="_blank"
@@ -1133,11 +1048,11 @@
                            <div
                              className="euiTextColor euiTextColor--subdued"
                            >
                              <p>
                                <FormattedMessage
-                                 defaultMessage="By default, a request fails if any of the queried remote clusters are unavailable. To continue sending a request to other remote clusters if this cluster is unavailable, enable {optionName}. {learnMoreLink}"
+                                 defaultMessage="A request fails if any of the queried remote clusters are unavailable. To send requests to other remote clusters if this cluster is unavailable, enable {optionName}. {learnMoreLink}"
                                  id="xpack.remoteClusters.remoteClusterForm.sectionSkipUnavailableDescription"
                                  values={
                                    Object {
                                      "learnMoreLink": <ForwardRef
                                        target="_blank"
@@ -1156,11 +1071,11 @@
                                        />
                                      </strong>,
                                    }
                                  }
                                >
-                                 By default, a request fails if any of the queried remote clusters are unavailable. To continue sending a request to other remote clusters if this cluster is unavailable, enable 
+                                 A request fails if any of the queried remote clusters are unavailable. To send requests to other remote clusters if this cluster is unavailable, enable 
                                  <strong>
                                    <FormattedMessage
                                      defaultMessage="Skip if unavailable"
                                      id="xpack.remoteClusters.remoteClusterForm.sectionSkipUnavailableDescription.optionNameLabel"
                                      values={Object {}}
    at Object.test (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/plugins/remote_clusters/public/application/sections/components/remote_cluster_form/remote_cluster_form.test.js:30:25)
    at Object.asyncJestTest (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
```
</details>

---

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/","number":"33812","commit":"79e41e31def49ccc85257edd1eb91a6067ba291a"}]}
PIPELINE-->

gchaps commented:
**Intro sentence**

Add a remote cluster that connects to seed nodes or a single proxy request.

**Proxy address**

The address to use for remote connections.

**Node connections**

The number of gateway nodes to use for requests.  (Not sure about this, but want to get away from ending with the word "to")

**Tooltip**:

Edit this cluster to update **the** deprecated settings.

**Server name (optional)**

- Add optional to label
- Would it be good to include an example as placeholder text
- Label is server name but talks about hostname
- Let's discuss some ways to reduce the text

alisonelizabeth commented:
@gchaps thanks for the review! I've updated the code per your suggestions.

> The number of gateway nodes to use for requests. (Not sure about this, but want to get away from ending with the word "to")

I changed this to: "The number of gateway nodes to connect to for this cluster." WDYT?

> Server name (optional)
> Add optional to label
> Would it be good to include an example as placeholder text
> Label is server name but talks about hostname
> Let's discuss some ways to reduce the text

I added `(optional)` to the label. I also shortened the text a little bit, but could probably still be improved. Here is the ES documentation on it: https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-remote-clusters.html#remote-cluster-proxy-settings. What do you think about linking to this in the help text?

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34172/)
* Commit: 5e6afb78866f79ff36787216fd11fe33f0e0175a

### History
* :broken_heart: [Build #33812](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/) failed 79e41e31def49ccc85257edd1eb91a6067ba291a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/","number":"33812","commit":"79e41e31def49ccc85257edd1eb91a6067ba291a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34172/","number":"34172","commit":"5e6afb78866f79ff36787216fd11fe33f0e0175a"}]}
PIPELINE-->

gchaps commented:
Good suggestions, @alisonelizabeth.

I like this text:

The number of gateway nodes to connect to for this cluster.

I shortened this text a bit and added the "Learn more" link.

A string sent in the server_name field of the TLS Server Name Indication extension if TLS is enabled. Learn more.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34332/)
* Commit: 72649df722a861f6c8b8efe1b94df4a56a7cf474

### History
* :green_heart: [Build #34172](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34172/) succeeded 5e6afb78866f79ff36787216fd11fe33f0e0175a
* :broken_heart: [Build #33812](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/) failed 79e41e31def49ccc85257edd1eb91a6067ba291a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33812/","number":"33812","commit":"79e41e31def49ccc85257edd1eb91a6067ba291a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34172/","number":"34172","commit":"5e6afb78866f79ff36787216fd11fe33f0e0175a"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34332/","number":"34332","commit":"72649df722a861f6c8b8efe1b94df4a56a7cf474"}]}
PIPELINE-->
